### PR TITLE
Configure "live" branches for stack books

### PR DESCRIPTION
**Pull request by nik9000**
Configures which branches are "live" for book that are versioned with
the stack.

### CODE REVIEW

**Comment by nik9000** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

File: conf.yaml [hunk 1]
```diff
@@ -95,6 +95,7 @@ contents:
             current:    7.4
             index:      docs/en/getting-started/index.asciidoc
             branches:   [ master, 7.x, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3 ]
+            live:       *stacklive
```
Comment (lcawl) [inline conf.yaml]: Does this setting mean that this book uses the same values as are specified in &stacklive for the Installation and Upgrade Guide?  If so, that's a lot easier than copying the same values over and over.

File: conf.yaml [hunk 2]
```diff
@@ -1253,6 +1274,7 @@ contents:
                 index:      docs/guide/index.asciidoc
                 current:    7.4
                 branches:   [ master, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0 ]
+                live:       *stacklive
```
Comment (bmorelli25) [inline conf.yaml]: APM doesn't build `7.x` documentation. Will using `*stacklive` here (which lists `7.x` as a live branch) cause a problem?

File: conf.yaml [hunk 3]
```diff
@@ -1269,6 +1291,7 @@ contents:
                 index:      docs/index.asciidoc
                 current:    7.4
                 branches:   [ master, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0 ]
+                live:       *stacklive
```
Comment (bmorelli25) [inline conf.yaml]: Same question as above

File: conf.yaml (hunk 2 above)
Comment (nik9000) [reply conf.yaml]: It *ought* to be fine if the `live` list is "bigger" than the branches list.

File: conf.yaml (hunk 1 above)
Comment (nik9000) [reply conf.yaml]: It does.

**Pull request merged**